packages/cli/src/mud.ts

```diff
@@ -1,8 +1,13 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-esm
 
+// -S ts-node-esm --no-warnings
+// TODO env -S flag may not be well supported, gather feedback
```

Comment:
what does the flag do?

Reply:
I'm not sure if I need it now, but if there're `Experimental warnings` in console, this flag could remove them. Keeping it in comments to not look for it later